orca-web/src/main/groovy/com/netflix/spinnaker/orca/Main.groovy

```diff
@@ -55,6 +56,7 @@ import org.springframework.scheduling.annotation.EnableAsync
 @EnableAutoConfiguration(exclude = [BatchAutoConfiguration, GroovyTemplateAutoConfiguration])
 @EnableBatchProcessing(modular = true)
 @Import([
+  PlatformComponents,
```

Comment:
needed this early in the import list or `@ConditionalOnBean(DiscoveryClient.class)` wasn't evaluating properly (previously we were getting this via autoconfiguration)